doc/getting-started/building-from-source.rst

```diff
@@ -176,99 +182,51 @@ Assuming that you are at the top level of the acrn-hypervisor directory, perform
 
   .. code-block:: none
 
-    $ make all BOARD_FILE=$PWD/misc/vm_configs/xmls/board-xmls/nuc6cayh.xml \
-    SCENARIO_FILE=$PWD/misc/vm_configs/xmls/config-xmls/nuc6cayh/sdc.xml
+    $ make all BOARD=$PWD/misc/vm_configs/xmls/board-xmls/nuc6cayh.xml \
+      SCENARIO=$PWD/misc/vm_configs/xmls/config-xmls/nuc6cayh/sdc.xml
 
+.. note::
+   ACRN uses XML files to summarize board characteristics and scenario
+   settings. The ``BOARD`` and ``SCENARIO`` variables accept board/scenario
+   names as well as paths to XML files. When board/scenario names are given, the
+   build system searches for XML files with the same names under
+   ``misc/config_tools/data/``. When relative files paths are given, the build
```

Comment:
I would like that we enhance this explanation a bit, how about: `When paths (absolute or relative) to the XML files are given, the build system uses the files pointed at. If relative paths are used, they are considered relative to the root acrn-hypervisor` folder.

Reply:
Sure. I've integrated your suggestion in the refined version. BTW, relative paths are considered relative to the current working directory.